univocity-trader-chart/src/main/java/com/univocity/trader/chart/indicators/DefaultIndicators.java

```diff
@@ -164,6 +164,11 @@ public static MovingAverage MovingAverage(@PositiveDefault(12) int length, TimeI
 		return Indicators.MovingAverage(length, interval, valueGetter(valueGetter));
 	}
 
+	@Overlay
```

Comment:
The overlay annotation should not be used for this indicator as the values of these indicator are too far away from the candle prices. Removing it will make the chart render properly